sopel/coretasks.py

```diff
@@ -599,6 +602,14 @@ def _parse_modes(bot, args):
             _send_who(bot, channel_name)
             return
 
+    if param_idx != len(params):
+        LOGGER.warning(
+            "%d extra arguments received for MODE: args=%r chanmodes=%r",
+            len(params) - param_idx,
```

Comment:
To answer the question about `1 arguments`, kinda yeah.

```suggestion
    if param_idx != len(params):
        extra = len(params) - param_idx
        LOGGER.warning(
            "%d extra argument%s received for MODE: args=%r chanmodes=%r",
            extra,
            's' if extra != 1 else '',
```

Reply:
We can figure that out from the other info provided, so I'll just keep it simple (and make translations easier).